README.md

```diff
@@ -0,0 +1,30 @@
+# CytoCurve
+
+## Welcome to CytoCurve
+CytoCurve is an opensource sowftware that provides a set of tools for the selection of representative images from a set of images.
+
+## Functions of CytoCurve
+* **Image selection**: CytoCurve will select a representative image from a set of images.
+* **Image Distribution Analysis**: Choose your own representative image and CytoCurve will graph where in the image set distribution your image is located.
+
+
+
+
+# Installing and Using CytoCurve
```

Comment:
Consider using a header 2 `##` here to keep the header pattern consistent, "nest" the header 3's (`###`) below in sequence, and avoid more than one header 1 (it's a general best practice to include only one header 1 per document).